### ISSUE TITLE
Error Flet build apk

### ISSUE THREAD

**Issue by Ambassador-of-programming** [flet-dev/flet#2514]
My os Parrot OS

flet build apk
Creating Flutter bootstrap project...OK
Customizing app icons and splash images...OK
Generating app icons...OK
Generating splash screens...OK
Packaging Python app...OK
Building .apk for Android...Resolving dependencies...
  ffi 2.1.0 (2.1.2 available)
  fl_chart 0.65.0 (0.66.1 available)
  flutter_lints 2.0.3 (3.0.1 available)
  flutter_native_splash 2.3.7 (2.3.10 available)
  js 0.6.7 (0.7.0 available)
  lints 2.1.1 (3.0.0 available)
  matcher 0.12.16 (0.12.16+1 available)
  material_color_utilities 0.5.0 (0.8.0 available)
  meta 1.10.0 (1.11.0 available)
  package_info_plus 4.2.0 (5.0.1 available)
  path 1.8.3 (1.9.0 available)
  petitparser 5.4.0 (6.0.2 available)
  test_api 0.6.1 (0.7.0 available)
  toml 0.14.0 (0.15.0 available)
  web 0.3.0 (0.4.2 available)
  web_socket_channel 2.4.0 (2.4.3 available)
  xml 6.3.0 (6.5.0 available)
Got dependencies!
17 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.

Running Gradle task 'assembleRelease'...                        
Checking the license for package NDK (Side by side) 23.1.7779620 in /home/patron/Android/Sdk/licenses
License for package NDK (Side by side) 23.1.7779620 accepted.
Preparing "Install NDK (Side by side) 23.1.7779620 (revision: 23.1.7779620)".
Warning: An error occurred while preparing SDK package NDK (Side by side) 23.1.7779620: invalid stored
block lengths.:
java.util.zip.ZipException: invalid stored block lengths
        at java.base/java.util.zip.InflaterInputStream.read(InflaterInputStream.java:164)
        at 
org.apache.commons.compress.archivers.zip.InflaterInputStreamWithStatistics.read(InflaterInputStreamWi
thStatistics.java:67)
        at java.base/java.io.BufferedInputStream.read1(BufferedInputStream.java:282)
        at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:343)
        at java.base/java.io.FilterInputStream.read(FilterInputStream.java:106)
        at com.android.repository.util.InstallerUtil.readZipEntry(InstallerUtil.java:186)
        at com.android.repository.util.InstallerUtil.unzip(InstallerUtil.java:143)
        at com.android.repository.impl.installer.BasicInstaller.doPrepare(BasicInstaller.java:90)
        at 
com.android.repository.impl.installer.AbstractPackageOperation.prepare(AbstractPackageOperation.java:3
35)
        at com.android.builder.sdk.DefaultSdkLoader.installRemotePackages(DefaultSdkLoader.java:365)
        at com.android.builder.sdk.DefaultSdkLoader.installSdkTool(DefaultSdkLoader.java:454)
        at com.android.build.gradle.internal.SdkHandler.installNdk(SdkHandler.java:322)
        at 
com.android.build.gradle.internal.cxx.configure.NdkLocatorKt.findNdkPathImpl(NdkLocator.kt:231)
        at 
com.android.build.gradle.internal.cxx.configure.NdkLocatorKt.findNdkPathImpl(NdkLocator.kt:423)
        at com.android.build.gradle.internal.cxx.configure.NdkLocator.findNdkPath(NdkLocator.kt:455)
        at com.android.build.gradle.internal.ndk.NdkHandler.getNdkStatus(NdkHandler.kt:85)
        at com.android.build.gradle.internal.ndk.NdkHandler.getNdkPlatform(NdkHandler.kt:98)
        at 
com.android.build.gradle.internal.cxx.gradle.generator.CxxConfigurationModelKt.tryCreateConfigurationP
arameters(CxxConfigurationModel.kt:211)
        at 
com.android.build.gradle.internal.cxx.configure.CxxCreateGradleTasksKt.createCxxVariantBuildTask(CxxCr
eateGradleTasks.kt:84)
        at 
com.android.build.gradle.internal.LibraryTaskManager.doCreateTasksForVariant(LibraryTaskManager.java:2
17)
        at com.android.build.gradle.internal.TaskManager.createTasksForVariant(TaskManager.kt:432)
        at com.android.build.gradle.internal.TaskManager.createTasks(TaskManager.kt:327)
        at com.android.build.gradle.internal.plugins.BasePlugin.createAndroidTasks(BasePlugin.kt:693)
        at 
com.android.build.gradle.internal.plugins.BasePlugin$createTasks$2$1.call(BasePlugin.kt:572)
        at 
com.android.build.gradle.internal.profile.NoOpAnalyticsConfiguratorService.recordBlock(NoOpAnalyticsCo
nfiguratorService.kt:53)
        at 
com.android.build.gradle.internal.plugins.BasePlugin$createTasks$2.accept(BasePlugin.kt:567)
        at 
com.android.build.gradle.internal.plugins.BasePlugin$createTasks$2.accept(BasePlugin.kt:564)
        at 
com.android.build.gradle.internal.crash.CrashReporting$afterEvaluate$1.execute(crash_reporting.kt:37)
        at 
com.android.build.gradle.internal.crash.CrashReporting$afterEvaluate$1.execute(crash_reporting.kt:35)
        at 
org.gradle.configuration.internal.DefaultUserCodeApplicationContext$CurrentApplication$1.execute(Defau
ltUserCodeApplicationContext.java:123)
        at 
org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingAction$
1.run(DefaultListenerBuildOperationDecorator.java:171)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
29)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
26)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
66)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
59)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:15
7)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59
)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
        at 
org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68
)
        at 
org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingAction.
execute(DefaultListenerBuildOperationDecorator.java:168)
        at 
org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:95
)
        at 
org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:83
)
        at 
org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:43)
        at 
org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:245)
        at 
org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:157)
        at 
org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:61)
        at 
org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:346)
        at 
org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:249)
        at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:141)
        at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
        at 
org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAda
pter.java:94)
        at jdk.proxy1/jdk.proxy1.$Proxy42.afterEvaluate(Unknown Source)
        at 
org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate$1.execute(LifecycleProj
ectEvaluator.java:247)
        at 
org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate$1.execute(LifecycleProj
ectEvaluator.java:244)
        at 
org.gradle.api.internal.project.DefaultProject.stepEvaluationListener(DefaultProject.java:1434)
        at 
org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate.run(LifecycleProjectEva
luator.java:253)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
29)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
26)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
66)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
59)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:15
7)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59
)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
        at 
org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68
)
        at 
org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.lambda$run$0(LifecycleProje
ctEvaluator.java:114)
        at 
org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.lambda$applyToMutableStat
e$0(DefaultProjectStateRegistry.java:351)
        at 
org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.lambda$fromMutableState$1
(DefaultProjectStateRegistry.java:374)
        at 
org.gradle.internal.work.DefaultWorkerLeaseService.withReplacedLocks(DefaultWorkerLeaseService.java:34
5)
        at 
org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.fromMutableState(DefaultP
rojectStateRegistry.java:374)
        at 
org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.applyToMutableState(Defau
ltProjectStateRegistry.java:350)
        at 
org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.run(LifecycleProjectEvaluat
or.java:100)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
29)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
26)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
66)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
59)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:15
7)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59
)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
        at 
org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68
)
        at 
org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:72)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:761)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:152)
        at 
org.gradle.api.internal.project.ProjectLifecycleController.lambda$ensureSelfConfigured$1(ProjectLifecy
cleController.java:63)
        at 
org.gradle.internal.model.StateTransitionController.lambda$doTransition$12(StateTransitionController.j
ava:236)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:247)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:235)
        at 
org.gradle.internal.model.StateTransitionController.lambda$maybeTransitionIfNotCurrentlyTransitioning$
9(StateTransitionController.java:196)
        at org.gradle.internal.work.DefaultSynchronizer.withLock(DefaultSynchronizer.java:34)
        at 
org.gradle.internal.model.StateTransitionController.maybeTransitionIfNotCurrentlyTransitioning(StateTr
ansitionController.java:192)
        at 
org.gradle.api.internal.project.ProjectLifecycleController.ensureSelfConfigured(ProjectLifecycleContro
ller.java:63)
        at 
org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.ensureConfigured(DefaultP
rojectStateRegistry.java:325)
        at org.gradle.execution.TaskPathProjectEvaluator.configure(TaskPathProjectEvaluator.java:33)
        at 
org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:49)
        at 
org.gradle.configuration.DefaultProjectsPreparer.prepareProjects(DefaultProjectsPreparer.java:50)
        at 
org.gradle.configuration.BuildTreePreparingProjectsPreparer.prepareProjects(BuildTreePreparingProjects
Preparer.java:64)
        at 
org.gradle.configuration.BuildOperationFiringProjectsPreparer$ConfigureBuild.run(BuildOperationFiringP
rojectsPreparer.java:52)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
29)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:
26)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
66)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
59)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:15
7)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59
)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:47)
        at 
org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:68
)
        at 
org.gradle.configuration.BuildOperationFiringProjectsPreparer.prepareProjects(BuildOperationFiringProj
ectsPreparer.java:40)
        at 
org.gradle.initialization.VintageBuildModelController.lambda$prepareProjects$3(VintageBuildModelContro
ller.java:89)
        at 
org.gradle.internal.model.StateTransitionController.lambda$doTransition$12(StateTransitionController.j
ava:236)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:247)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:235)
        at 
org.gradle.internal.model.StateTransitionController.lambda$transitionIfNotPreviously$10(StateTransitio
nController.java:210)
        at org.gradle.internal.work.DefaultSynchronizer.withLock(DefaultSynchronizer.java:34)
        at 
org.gradle.internal.model.StateTransitionController.transitionIfNotPreviously(StateTransitionControlle
r.java:206)
        at 
org.gradle.initialization.VintageBuildModelController.prepareProjects(VintageBuildModelController.java
:89)
        at 
org.gradle.initialization.VintageBuildModelController.prepareToScheduleTasks(VintageBuildModelControll
er.java:71)
        at 
org.gradle.internal.build.DefaultBuildLifecycleController.lambda$prepareToScheduleTasks$2(DefaultBuild
LifecycleController.java:127)
        at 
org.gradle.internal.model.StateTransitionController.lambda$doTransition$12(StateTransitionController.j
ava:236)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:247)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:235)
        at 
org.gradle.internal.model.StateTransitionController.lambda$maybeTransition$8(StateTransitionController
.java:187)
        at org.gradle.internal.work.DefaultSynchronizer.withLock(DefaultSynchronizer.java:34)
        at 
org.gradle.internal.model.StateTransitionController.maybeTransition(StateTransitionController.java:183
)
        at 
org.gradle.internal.build.DefaultBuildLifecycleController.prepareToScheduleTasks(DefaultBuildLifecycle
Controller.java:125)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeWorkPreparer.scheduleRequestedTasks(DefaultBuildTreeWork
Preparer.java:33)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$doScheduleAndRunTasks$2(Defau
ltBuildTreeLifecycleController.java:89)
        at 
org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.withNewWorkGraph(DefaultIncludedBuildTaskG
raph.java:101)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.doScheduleAndRunTasks(DefaultBuildTr
eeLifecycleController.java:88)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$runBuild$4(DefaultBuildTreeLi
fecycleController.java:106)
        at 
org.gradle.internal.model.StateTransitionController.lambda$transition$5(StateTransitionController.java
:166)
        at 
org.gradle.internal.model.StateTransitionController.doTransition(StateTransitionController.java:247)
        at 
org.gradle.internal.model.StateTransitionController.lambda$transition$6(StateTransitionController.java
:166)
        at org.gradle.internal.work.DefaultSynchronizer.withLock(DefaultSynchronizer.java:44)
        at 
org.gradle.internal.model.StateTransitionController.transition(StateTransitionController.java:166)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.runBuild(DefaultBuildTreeLifecycleCo
ntroller.java:103)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.scheduleAndRunTasks(DefaultBuildTree
LifecycleController.java:69)
        at 
org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:31)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
        at 
org.gradle.internal.buildtree.ProblemReportingBuildActionRunner.run(ProblemReportingBuildActionRunner.
java:49)
        at 
org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRu
nner.java:65)
        at 
org.gradle.tooling.internal.provider.FileSystemWatchingBuildActionRunner.run(FileSystemWatchingBuildAc
tionRunner.java:136)
        at 
org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildAc
tionRunner.java:41)
        at 
org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.lambda$execute$0(RootBuildLifecycleBuil
dActionExecutor.java:40)
        at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:122)
        at 
org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.execute(RootBuildLifecycleBuildActionEx
ecutor.java:40)
        at 
org.gradle.internal.buildtree.DefaultBuildTreeContext.execute(DefaultBuildTreeContext.java:40)
        at 
org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.lambda$execute$0(BuildTreeLifecycleBuil
dActionExecutor.java:65)
        at org.gradle.internal.buildtree.BuildTreeState.run(BuildTreeState.java:53)
        at 
org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.execute(BuildTreeLifecycleBuildActionEx
ecutor.java:65)
        at 
org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionE
xecutor.java:61)
        at 
org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionE
xecutor.java:57)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(Defaul
tBuildOperationRunner.java:204)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(Defaul
tBuildOperationRunner.java:199)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
66)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:
59)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:15
7)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59
)
        at 
org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
        at 
org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:7
3)
        at 
org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor.execute(RunAsBuildOperationBuildAction
Executor.java:57)
        at 
org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.lambda$execute$0(RunAsWorkerThreadBuildA
ctionExecutor.java:36)
        at 
org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:249)
        at 
org.gradle.internal.work.DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:10
9)
        at 
org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.execute(RunAsWorkerThreadBuildActionExec
utor.java:36)
        at 
org.gradle.tooling.internal.provider.continuous.ContinuousBuildActionExecutor.execute(ContinuousBuildA
ctionExecutor.java:110)
        at 
org.gradle.tooling.internal.provider.SubscribableBuildActionExecutor.execute(SubscribableBuildActionEx
ecutor.java:64)
        at 
org.gradle.internal.session.DefaultBuildSessionContext.execute(DefaultBuildSessionContext.java:46)
        at 
org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter$ActionImpl.apply(BuildSe
ssionLifecycleBuildActionExecuter.java:100)
        at 
org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter$ActionImpl.apply(BuildSe
ssionLifecycleBuildActionExecuter.java:88)
        at org.gradle.internal.session.BuildSessionState.run(BuildSessionState.java:69)
        at 
org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLife
cycleBuildActionExecuter.java:62)
        at 
org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLife
cycleBuildActionExecuter.java:41)
        at 
org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidating
ActionExecuter.java:63)
        at 
org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidating
ActionExecuter.java:31)
        at 
org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureRepor
tingActionExecuter.java:52)
        at 
org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureRepor
tingActionExecuter.java:40)
        at 
org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.jav
a:47)
        at 
org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.jav
a:31)
        at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:65)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at 
org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at 
org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:29)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at 
org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDa
emon.java:35)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at 
org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)
        at 
org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)
        at org.gradle.util.internal.Swapper.swap(Swapper.java:38)
        at 
org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at 
org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.jav
a:84)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at 
org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at 
org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.j
ava:52)
        at 
org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)
        at 
org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
"Install NDK (Side by side) 23.1.7779620 (revision: 23.1.7779620)" failed.
Running Gradle task 'assembleRelease'...                         2139.1s

[CXX1101] NDK at /home/patron/Android/Sdk/ndk/23.1.7779620 did not have a source.properties file

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':serious_python_android'.
> com.android.builder.sdk.InstallFailedException: Failed to install the following SDK components:
      ndk;23.1.7779620 NDK (Side by side) 23.1.7779620
  Install the missing components using the SDK manager in Android Studio.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 35m 38s
Gradle task assembleRelease failed with exit code 1

Error building Flet app - see the log of failed command above.

**Comment by lekshmanmj** [comment]
please install ndk using android-studio's sdkmanager
![image](https://github.com/flet-dev/flet/assets/35882740/2e10811c-ae11-46f7-a9ad-5ebbd2cd5d2b)

**Comment by FeodorFitsner** [comment]
It is fixed in Flet 0.23.1.

Please cleanup Cookiecutter cache before running `flet build` command.

On Windows:

```
rd /s /q %USERPROFILE%\.cookiecutters
```

On Mac/Linux:

```
rm -rf ~/.cookiecutters
```
